pynest/nest/lib/hl_api_subnets.py

```diff
@@ -236,6 +244,7 @@ def GetNetwork(gid, depth):
 
 
 @check_stack
+@deprecated("", "BeginSubnet: no longer needed, work with GIDCollections")
```

Comment:
Better: "BeginSubnet is deprecated and will be removed in NEST3. See documentation for details."